classes/LineManagerNotifier.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace Claromentis\ThankYou;
+
+use ErrorHandler;
+use IMessage;
+use NotificationMessage;
+use User;
+
+class LineManagerNotifier extends ErrorHandler
```

Comment:
Just realised that this extends `ErrorHandler`. Does it need to? I'd recommend avoiding this pattern.

Reply:
This has been resolved, cheers